RELEASE-LEADS.md

```diff
@@ -48,13 +48,15 @@ This roster is seeded with all approvers from Eventing workgroups.
 | v0.17   | 2020-08-18   | yanweiguo      | Harwayne        | -           | 2020-08-11 |
 | v0.18   | 2020-09-29   | ZhiminXiang    | n3wscott        | 2020-08-19  | 2020-09-22 |
 | v0.19   | 2020-11-10   | julz           | n3wscott        | 2020-09-30  | 2020-11-03 |
-| v0.20   | 2020-12-22   | nak3           | slinkydeveloper | 2020-11-11  | 2020-12-15 |
-| v0.21   | 2021-02-02   | mattmoor       | lionelvillard   | 2020-12-23  | 2021-01-26 |
-| v0.22   | 2021-03-16   | markusthoemmes | evankanderson   | 2020-02-03  | 2021-03-09 |
-| v0.23   | 2021-04-27   | tcnghia        | vaikas          | 2020-03-17  | 2021-04-20 |
-| v0.24   | 2021-06-08   | dprotaso       | matzew          | 2020-04-28  | 2021-06-01 |
-| v0.25   | 2021-07-20   | vagababov      | grantr          | 2020-06-09  | 2021-07-13 |
-| v0.26   | 2021-08-31   | JRBANCEL       | ...             | 2020-07-21  | 2021-08-24 |
+| v0.20   | 2021-01-05   | nak3           | slinkydeveloper | 2020-11-11  | 2020-12-15 |
```

Comment:
The week before a release, the release managers need to be hands on keyboards cutting branches, in the new world. I would push the release one more week to allow for this. Otherwise the release will not go smoothly.

Reply:
Hi @vagababov @grantr @tcnghia @evankanderson  I am tagging you as the schedule was decided by https://github.com/knative/community/pull/226.
Is it alright to push one more week for the v0.20 release schedule?

Reply:
Good question -- I have no intrinsic problem with doing this; the previous date extension was decided with the notion that a release (and subsequent hangover of e.g. docs and operator at the moment) should have hands on deck after release in case of bugs.

If we need hands on deck before release (and can't relax that by e.g. pre-cutting the `pkg` branch of other prep), then moving another week seems okay.

Reply:
Okay, pushed one more week.  
@slinkydeveloper  we will do pkg cut on `2020-01-05` and release on `2020-01-12`. Is that alright?

Reply:
Can we do pkg cut on `2020-01-07` so we're after the epiphany? I guess we won't have big contributions to pkg in that period, 5 days of freeze more than one week is fine i think. Or is that a "strong" requirement to be one week before?

Reply:
@slinkydeveloper Sure, I'm fine with `2020-01-07 `.  Could you send the PR to update the pkg cut schedule?